src/uwtools/tests/test_rocoto.py

```diff
@@ -0,0 +1,54 @@
+# pylint: disable=missing-function-docstring
+"""
+Tests for uwtools.rocoto module.
+"""
+from importlib import resources
+
+import yaml
+
+from uwtools import rocoto
+from uwtools.tests import support
+
+# pylint: disable=protected-access
```

Comment:
Please combine with line l:
```
# pylint: disable=missing-function-docstring,protected-accesss
```

Reply:
Just for the future in case someone sees this, it took me a second to realize there was an extra s on the end of access.
`# pylint: disable=missing-function-docstring,protected-access`